boards/arm/stm32f0_disco/doc/stm32f0_disco.rst

```diff
@@ -0,0 +1,141 @@
+.. _stm32f0_disco_board:
+
+ST STM32F0DISCOVERY
+################
+
+Overview
+********
+
+The STM32F0 Discovery development board with STM32F051R8T6 MCU
```

Comment:
Complete sentences are preferred.  Let's combine these into:

    The STM32F0 Discovery development board uses an STM32F051R8T6 MCU and
    integrates the ST-LINK/V2-1 debugger and programmer.  It also comes with a
    comprehensive STM32 software HAL library and various packaged software
    examples.